### DISCUSSION TITLE
Помощь - FAR2l (та, что вызывается клавишей F1)

### DISCUSSION THREAD

**Discussion by Mischutka** [elfmz/far2l#1249] [Q&A]
Почему-то оказались пустыми следующие разделы:

Индекс файла помощи/Запрос привилегий администратора;
Индекс файла помощи/Меню выбора плагина;
Индекс файла помощи/настройка автозавершения.

Содержимое этих разделов забыли перенести из версии FAR для MS Windows или же они пусты по какой-то другой причине?

**Comment by elfmz** [comment]
Ну, эти пункты нигде не используются и остались от исходной виндовой версии фара, где они тоже не использовались - заготовки на будущее наверное. Скорее всего их можно безболезненно удалить, но лучше вначале детально поразбираться с ними.

**Reply by elfmz** [reply]
Поправочка - пункты используются в исходниках, но пустые изначально. Удалять нельзя, надо наполнять)

**Comment by Mischutka** [comment]
Найдены опечатки:

Индекс файла помощи/Терминал: **Расширения терминала FAR2L** хотя FAR2L сам по себе это приложение _ткстового_ режима...

Индекс файла помощи/Дополнительные параметры поиска (4-й абзац снизу): В _отличии_ от режимов панелей...

**Comment by Mischutka** [comment]
Неуместное упоминание MS Windows, учитывая, что FAR2L — это версия FAR Manager для Linux:

Индекс файла помощи/Ассоциации файлов: Если для данного файла отсутствуют ассоциированные команды запуска и установлен параметр Использовать стандартные типы в Системных параметрах, то FAR2L пытается использовать ассоциации Windows для запуска этого типа файлов.

**Comment by Mischutka** [comment]
Индекс файла помощи/Жёсткие ссылки, связи папок, символические ссылки — неуместная привязка к MS Windows, учитывая, что FAR2L — это версия FAR Manager для Linux.

**Comment by Mischutka** [comment]
Индекс файла помощи/Команды управления панелями - сервисные команды: даны примеры для MS Windows, что некорректно, так как, FAR2L — это версия FAR Manager для Linux.

**Comment by Mischutka** [comment]
Индекс файла помощи/Команды операционной системы: приведены команды MS Windows вместо команд Linux.

**Comment by Mischutka** [comment]
Индекс файла помощи/Ошибка: Ошибка выполнения: неуместные отсылки к MS Windows.

**Comment by Mischutka** [comment]
Индекс файла помощи/Предупреждение: Исходный файл содержит более одного потока данных: отсылки к файловой системе NTFS и Windows Explorer.

**Comment by Mischutka** [comment]
Индекс файла помощи/Клавиши управления — Примечания п.2: отсылка к свойству MS Windows.

**Comment by Mischutka** [comment]
Индекс файла помощи/Раскраска файлов и группы сортировки: редактирование — последний абзац: дан перечень атрибутов, используемых на дисках с файловой системой NTFS, упомянут атрибут, который не используется в Windows 2000/XP/2003.

**Comment by Mischutka** [comment]
Индекс файла помощи/Режимы сортировки — в Замечаниях по числовой сортировке даны привязки к MS Windows XP И MS Windows 2000.

**Comment by Mischutka** [comment]
Индекс файла помощи/Создание папки — даны команды для MS Windows.

**Comment by Mischutka** [comment]
Индекс файла помощи/Список Hotplug-устройств — отсылка к MS Windows.

**Comment by Mischutka** [comment]
Индекс файла помощи/Формат командной строки — $@XX и далее: приведён пример для MS Windows.
